### ISSUE TITLE
Add command line option to skip integration dialog

### ISSUE THREAD

**Issue by LeSnake04** [TheAssassin/AppImageLauncher#439]
**Is your feature request related to a problem? Please describe.**
I am the maintainer of  https://aur.archlinux.org/packages/cpod-appimage and it annoys me that on launch it alwys ask if i want to ingrate the package, even though I made an proper integration. If i would integrate it even more, it would make an second entry in menu, which doesnt get updates. 

**Describe the solution you'd like**
It would be really nice if I could add an command line option to the .desktop file which disables the popup.

**Additional context**
Related: #111

**Comment by TheAssassin** [comment]
1. What's the goal of that package? Do you just ship an AppImage inside an Arch package?
1. Please provide a screenshot.
1. Please provide the AppImageLauncher version.

If my assumption from 1. is correct, we can discuss ways how to avoid the integration dialog.

**Comment by LeSnake04** [comment]
1. Yes, your assumbtion is correct. I did it to provide an way to get cpod working (the non appimage versions are currently broken) and while cpod techincally has an auto update function, https://aur.archlinux.org/packages/qmplay2-appimage (which i also maintain) doesnt have one
2. I am not sure from what you want the screenshot: the popup?

**Comment by TheAssassin** [comment]
"Suppressing" the dialog can be done, but I think that's not really necessary. Shipping the entire AppImage makes no sense in your case. How about `--appimage-extract`ing the contents and packaging the resulting AppDir? The result will be just as portable. AppImageUpdate and other advanced AppImage features (just like desktop integration) are most likely not needed in your case anyway?

**Comment by LeSnake04** [comment]
I tried it pand it turned out to be a bad idea for 2 reasons:
1. The package size grew by 300% (50MB to 150 mb)
2. It doesnt work since it tries to load the library's from the root /usr/share and AUR packages should NEVER EVER install third party libraries, since it will cause file conflicts if the real library is already installed or when the real library gets installed later

Thats why pretty much appimage based packages in the AUR  install the appimage directly

**Comment by LeSnake04** [comment]
Turned out some packages work with the extracted AppImage. I didn't took a closer look yet but I presume these packages can only do it because eighter:

1. the Appimage has no library packaged into it
2. They install the library from the repo/other AUR packages, making the install process way longer, may install versions incompatible with the app (Whats pretty much guarantied in old AppImages ) and risking the installation being broken if other AUR packages break.
3. It MAY be possible to install the packaged libraries without putting it into /usr/lib/ but I assume the difficulty of telling the system to not use the system library strongly varies between each program language and how the app is programmed.
I had problems with this in Python once: after I installed the program with pip to test my setup.py, Python refused to use the project files instead of the copy in the system path. Now I have to install it after every single change to the system path.
I am VERY sure this would be a lot of pain to set up especially in compiled and closed source applications


Long story short:
Its possible to use the extracted AppImage, but would require a lot of work to set up and maintain and defeats the Whole purpose of AppImages

You would do the AUR community a great favor by implementing this feature

**Comment by jpenguin** [comment]
`export APPIMAGELAUNCHER_DISABLE=true`

before the command is currently the only way

**Comment by LeSnake04** [comment]
> `export APPIMAGELAUNCHER_DISABLE=true`
> 
> before the command is currently the only way

Thanks for the tip! This is actually a good solution for my problem!

**Comment by LeSnake04** [comment]
> Turned out some packages work with the extracted AppImage. I didn't took a closer look yet but I asume these packages can only do it because eighter:
> 
> 1. the Appimage has no library packaged into it
> 2. They install the library from the repo/other AUR packages, making the install process way longer, may install versions incompatible with the app (Whats pretty much guarantied in old AppImages ) and risking the installation being broken if other packages break or get removed.
> 3. It MAY be possible to install the packaged libraries without putting it into /usr/lib/ but I assume the difficulty of telling the system to not use the system library strongly varies between each program language and how the app is programmed.
> I had problems with this in Python once: after I installed the program with pip to test my setup.py, Python refused to use the project files instead of the copy in the system path. Now I have to install it after every single change to the system path.
> I am VERY sure this would be a lot of pain to set up especially in compiled and closed source applications
> 
> 
> Long story short:
> Its possible to use the extracted AppImage, but would require a lot of work to set up and maintain and defeats the Whole purpose of AppImages
> 
> You would do the AUR community a great favor by implementing this feature
> 
>  

I just realise I can just use the AppRun from the extracted AppImage...

But the size point still stands.
